### DISCUSSION TITLE
API Specs with no examples

### DISCUSSION THREAD

**Discussion by saivamsilolla** [microcks/microcks#753] [Q&A] [answered]
Most of the Open API Specs(REST) we have are written without any examples. When I upload the Specs into microcks , I see that mock intance urls are not available. Is there a way mic rocks can generate responses for these specs ? Any idea to overcome this ?

**Comment by lbroudoux** [comment, marked as answer]
Hi @saivamsilolla,

Microcks cannot actually generate request and/or response examples for you. At the heart of the project, there's this belief that examples have incredible value for illustrating business expectations of your API and so we decided not to auto-generate examples for you. We foresee risks with examples auto-generation like having all your strings having the same 'lorem ipsum value' or all your integers having the `1` value. For us, this leads to meaningless and hard-to-understand mocks. We're targeting the exact opposite as described in our [Why Microcks?](https://microcks.io/blog/why-microcks/#1-business-requirements-without-translation) manifesto.

Hope it helps you understand the intent behind the project.
Cheers,
